app/views/organisations/_category_type.erb

```diff
@@ -7,7 +7,9 @@
     <% end %>
     <div class="span3 org-category">
       <%= ff.label :_destroy, class: "checkbox inline" do %>
-        <%= ff.check_box :_destroy, {:checked => ff.object.persisted?},  '0', '1' %>
+        <%= ff.check_box :_destroy, {:checked => @categories_selected.nil? || @categories_selected.empty? ? ff.object.persisted? : @categories_selected.index(ff.object.category.id.to_s).present?},  
```

Comment:
Actually it doesn't need to yet.  It was to deal with the case of a validation failure so that it remembered the boxes you had checked when the form reloaded.